Geometry/HcalTowerAlgo/src/HcalGeometry.cc

```diff
@@ -163,6 +161,23 @@ GlobalPoint HcalGeometry::getBackPosition(const DetId& id) const {
   }
 }
 
+CaloCellGeometry::CornersVec HcalGeometry::getCorners(const DetId& id) const {
```

Comment:
Yes - it was submitted when the earlier PR was not integrated and it was there in the earlier one